test/extended/testdata/router/router-grpc-interop-routes.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+parameters:
+- name: DOMAIN
+- name: TLS_CRT
+- name: TLS_KEY
+- name: TYPE
+objects:
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: grpc-interop-reencrypt
+    labels:
+      type: ${TYPE}
+  spec:
+    host: grpc-interop-reencrypt.${DOMAIN}
+    port:
+      targetPort: 8443
+    tls:
+      termination: reencrypt
+      insecureEdgeTerminationPolicy: Redirect
+      key: |-
+        ${TLS_KEY}
+      certificate: |-
+        ${TLS_CRT}
+    to:
+      kind: Service
+      name: grpc-interop
+      weight: 100
+    wildcardPolicy: None
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: grpc-interop-passthrough
+    labels:
+      type: ${TYPE}
+  spec:
+    host: grpc-interop-passthrough.${DOMAIN}
+    port:
+      targetPort: 8443
+    tls:
+      termination: passthrough
```

Comment:
No edge terminated route needed in this config?

Reply:
I simplified these tests to as little as possible taking place between the client call and backend. If we think it is worthwhile we add edge and reencrypt later.

Reply:
> No edge terminated route needed in this config?

Edge isn't supported. 

 This still exists but I removed the route: 

```go
				if routeType == routev1.TLSTerminationEdge {
					e2e.Logf("skipping %v tests - needs https://github.com/openshift/router/pull/104", routeType)
					continue
				}

```

I can put the edge route back.

Reply:
I put the edge route back.